ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -329,6 +345,12 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         }
 
         accessoryButton = !accessoryButtonTitle.isEmpty ? createAccessoryButton(withTitle: accessoryButtonTitle) : nil
+
+        /// `accessoryButton` and `accessoryView` occupy the same space at the trailing end of the view. If both are provided, the `accessoryView` will be given priority
+        if accessoryView != nil {
```

Comment:
Why do we check before setting this? What happens if we set `nil` here?

Reply:
The `accessoryButton` above uses the `accessoryView` to display itself. The `accessoryButton` won't be shown if we set `nil` here after setting a non-nil `accessoryButton`. `accessoryButton` setter for reference: 

```
    private var accessoryButton: UIButton? {
        didSet {
            accessoryView = accessoryButton
            if accessoryButton != nil {
                updateAccessoryButtonTitleStyle()
            }
        }
    }
  ```